Gopkg.toml

```diff
@@ -1,3 +1,7 @@
+required = [
+    "github.com/gogo/protobuf/proto",
```

Comment:
can't add a constraint on gogo because there is no go source in the root of this repo. 🤷‍♂️